### ISSUE TITLE
Please release a 2.7. setup.py install_requires not working 

### ISSUE THREAD

**Issue by k-funk** [SmileyChris/easy-thumbnails#536]
I don't use easy-thumbnails directly, but django-filer does (https://github.com/divio/django-filer/blob/master/setup.py#L12), and this library bumping to 2.6 broke my build. I tested, and `pip3 install easy-thumbnails==2.5.0` works for me, while `2.6.0` errors :

```
$ ./venv/bin/pip3 install easy-thumbnails==2.6.0
Collecting easy-thumbnails==2.6.0
  Using cached https://files.pythonhosted.org/packages/ae/37/442523964379e1076a4e9c29a89861f44e8c237fa6857e71b113cb2cb5bd/easy-thumbnails-2.6.tar.gz
    ERROR: Command errored out with exit status 1:
     command: /home/bggweb/bondage-a-go-go.com/bagg-web_6242209_2019-12-14-22-41-23/venv/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-n0cil94e/easy-thumbnails/setup.py'"'"'; __file__='"'"'/tmp/pip-install-n0cil94e/easy-thumbnails/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-n0cil94e/easy-thumbnails/pip-egg-info
         cwd: /tmp/pip-install-n0cil94e/easy-thumbnails/
    Complete output (1 lines):
    error in easy-thumbnails setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers; Expected ',' or end-of-list in django>=1.8,<2;python_version<"3" at ;python_version<"3"
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

Also confirmed that `pip install git+https://github.com/SmileyChris/easy-thumbnails` works, so a 2.7 bump will solve my issue.

Ubuntu 18.04.3 LTS
Python 3.5.0

**Comment by jrief** [comment]
Thanks for the reminder.
Just released [version 2.7](https://pypi.org/project/easy-thumbnails/).

**Comment by k-funk** [comment]
thank you! <3
